tests/test_git/test_git_init.py

```diff
@@ -0,0 +1,47 @@
+import sublime
+import os
+from .common import GSCreateRepoTestCase
+from GitSavvy.core import git_command
+
+
+def tidy_path(path):
+    return os.path.realpath(os.path.normcase(path))
+
+
+class TestInit(GSCreateRepoTestCase, git_command.GitCommand):
+
+    def test_01_window_path(self):
+        self.assertEqual(
+            tidy_path(self._temp_dir),
+            tidy_path(sublime.active_window().folders()[0]))
+        self.assertEqual(
+            tidy_path(self._temp_dir),
+            tidy_path(self.window.folders()[0]))
+
+    def test_02_add_file(self):
```

Comment:
I assume the numbering means you want test 01 to execute before test 02? This means these [tests are not isolated][1] and is probably not a good idea as dependencies between unit tests usually points at a problem.

[1]: https://pragprog.com/magazines/2012-01/unit-tests-are-first